tcms/kiwi_auth/tests/test_backends.py

```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
+from django.contrib.auth.models import AnonymousUser
+from django.test.utils import override_settings
+
+
+@override_settings(AUTHENTICATION_BACKENDS=[
```

Comment:
Using modify_settings here would be better suited b/c you can only append/prepend or remove desired settings instead of duplicating the rest https://docs.djangoproject.com/en/3.1/topics/testing/tools/